mula/tests/factories/boefje.py

```diff
@@ -12,9 +13,6 @@ class Meta:
     id: str = Faker("uuid4")
     name: str = Faker("name")
     description: str = Faker("text")
-    scan_level: int = fuzzy.FuzzyInteger(0, 4)
-    consumes: list[str] = LazyFunction(lambda: [])
-    produces: list[str] = LazyFunction(lambda: [])
 
```

Comment:
Cleanup? Or unrelated changes?

Produces is never relevant for the scheduler I'd say, and is mainly information to the user anyway.

Reply:
clean up